lib/geo/postgis.ex

```diff
@@ -28,7 +28,11 @@ if Code.ensure_loaded?(Ecto.Query) do
     end
 
     defmacro st_distance_sphere(geometryA, geometryB) do
-      quote do: fragment("ST_Distance_Sphere(?,?)", unquote(geometryA), unquote(geometryB))
+      quote do: fragment("ST_DistanceSphere(?,?)", unquote(geometryA), unquote(geometryB))
```

Comment:
Perhaps we leave the old macro? Folks will get the deprecation warning which should lead them to the new macro... I don't have an opinion on it.